app/serializers/api/v1/stockit_item_serializer.rb

```diff
@@ -31,6 +32,30 @@ def include_order?
 
     alias_method :include_designation_id?, :include_order?
 
+    def include_added_quantity?
+      @options[:include_added_quantity]
+    end
+
+    def include_in_hand_quantity?
+      @options[:include_in_hand_quantity]
+    end
+
+    def in_hand_quantity
```

Comment:
```suggestion
def on_hand_quantity
```

Reply:
Note: Currently working on adding counter cache columns to the package table. Which include `on_hand_quantity`, so I'll remove those methods once that's done : )

Reply:
Sure 👍